alot/buffers/search.py

```diff
@@ -50,6 +52,11 @@ def rebuild(self, reverse=False):
         else:
             order = self.sort_order
 
+        if self.threadlist:
+            selected_thread = self.get_selected_thread()
+        else:
```

Comment:
Is this else clause really needed? isnt `selected_thread == None` otherwise?

Reply:
Without the `else` statement, the variable would potentially be referenced without prior assignment.

One way to rewrite this hunk is declare `selected_thread = None` at the top of the function, and then we can remove the `else`.

Reply:
that's what I meant, thanks